pkg/authenticator/k8s/authenticator.go

```diff
@@ -423,3 +424,17 @@ func consumeInjectClientCertError(path string) string {
 
 	return string(content)
 }
+
+func logIPS(host string) {
+	ips, lookUpError := net.LookupIP(host)
+	if lookUpError != nil {
+		fmt.Fprintf(os.Stderr, "Could not get IPs: %v\n", lookUpError)
+		os.Exit(1)
+	}
+	ipsList := " IP Addresses: "
```

Comment:
There is a join function of any other allegant string function that can do what you doing here